### ISSUE TITLE
Task view button toggle works but not button itself in 22471.1000

### ISSUE THREAD

**Issue by Gaurav-Original-ClassicShellTester** [valinet/ExplorerPatcher#118]
Show Task view button toggle works but the Task view button itself that appears does nothing for me in Windows 11 22471.1000. Win+Tab works so Task View itself is functional. EP version is 22000.258.30.2.

With or without StartAllBack enabled so I made sure it was not interfering or breaking it.

Let me know if I need to provide any additional info.

**Comment by valinet** [comment]
Yeah.. again, try to see whether you can locate, or maybe someone already extracted those 4 icons. Then, I can make 4 EXEs that will look so sweet when pined to the taskbar and just take care of this basic job.

**Comment by Anix-x** [comment]
![image](https://user-images.githubusercontent.com/92230793/137953719-eca34908-def0-402e-aa00-957771850ff2.png)
Second from the right.
